Buildings/Controls/OBC/FDE/PackagedRTUs/OperatingMode.mo

```diff
@@ -0,0 +1,264 @@
+within Buildings.Controls.OBC.FDE.PackagedRTUs;
+block OperatingMode "Determine occupied or setback operating mode"
+
+  parameter Real TUpcntM(
+    min=0.1,
+    max=0.9,
+    final unit="1")=0.15
+    "Minimum decimal percentage of terminal unit requests required for mode change";
+
+  // --- inputs ---
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput occ
+    "True when occupied mode is active"
+    annotation (Placement(transformation(extent={{-160,40},{-120,80}}),
+        iconTransformation(extent={{-140,40},{-100,80}})));
+  Buildings.Controls.OBC.CDL.Interfaces.IntegerInput occReq
+    "Terminal unit occupancy requests"
+    annotation (Placement(transformation(extent={{-160,10},{-120,50}}),
+        iconTransformation(extent={{-140,10},{-100,50}})));
+  Buildings.Controls.OBC.CDL.Interfaces.IntegerInput TotalTU
+    "Total number of terminal units"
+    annotation (Placement(transformation(extent={{-160,-30},{-120,10}}),
+        iconTransformation(extent={{-140,-20},{-100,20}})));
+   Buildings.Controls.OBC.CDL.Interfaces.IntegerInput sbcReq
+    "Terminal unit setback cooling requests"
+    annotation (Placement(transformation(extent={{-160,-60},{-120,-20}}),
+        iconTransformation(extent={{-140,-50},{-100,-10}})));
+  Buildings.Controls.OBC.CDL.Interfaces.IntegerInput sbhReq
+    "Terminal unit setback heating requests" annotation (Placement(
+        transformation(extent={{-160,-98},{-120,-58}}), iconTransformation(
+          extent={{-140,-80},{-100,-40}})));
+  // --- outputs ---
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput yOcc
+    "True when occupied mode is active"
+    annotation (Placement(transformation(extent={{140,34},{180,74}}),
+        iconTransformation(extent={{100,34},{140,74}})));
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput ySBC
+    "True when setback cooling mode is active"
+    annotation (Placement(transformation(extent={{140,-48},{180,-8}}),
+        iconTransformation(extent={{100,-20},{140,20}})));
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput ySBH
+    "True when setback heating mode is active"
+    annotation (Placement(transformation(extent={{140,-90},{180,-50}}),
+        iconTransformation(extent={{100,-70},{140,-30}})));
+
+  Buildings.Controls.OBC.CDL.Logical.Or or2
```

Comment:
make sure there is comment to all the instances.

Reply:
I don't understand. Do you mean provide a comment for the logical Or instance? 'This is an OR block'

Reply:
exactly, we usually write "Logical or" if nothing more informative seems needed based on the logic. It is a duplication, I agree, but CDL requires all instances to be commented. You can be more descriptive in a way that you provide details on function of the instance.